ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/cco-manual-workload-identity/auth-with-sa/deprovision/cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity-auth-with-sa-deprovision-chain.yaml

```diff
@@ -0,0 +1,15 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity-auth-with-sa-deprovision
+  steps:
+  - chain: gather-network
+  - chain: gather-core-dump
+  - ref: gather-gcp-console
+  - ref: gather-proxy
```

Comment:
I guess we do not need the `gather-proxy` step, in this install, we do not involve proxy.